### DISCUSSION TITLE
All icons gone

### DISCUSSION THREAD

**Discussion by Nevarro** [hugo-fixit/FixIt#501] [General]
I'm sorry to have to ask once again a newbie question:

I don't think I have changed anything, when it comes to icons. However, icons everywhere have disappeared or, to be more precise, have been replaced with boxes.

Did I set an option by mistake?

<img width="409" alt="SCR-20240904-ilfs" src="https://github.com/user-attachments/assets/b2514f1f-1bd3-4970-8afc-c727243ddd74">

**Comment by Lruihao** [comment]
Maybe cause of low network.

**Reply by Nevarro** [reply]
I don't think so. It also looks that way when running hugo server locally.
